Tweetinvi.Logic/DTO/MessageUser.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using Tweetinvi.Logic.JsonConverters;
+using Tweetinvi.Models;
+using Tweetinvi.Models.DTO;
+using Tweetinvi.Models.Entities;
+
+namespace Tweetinvi.Logic.DTO
+{
+    public class MessageUser : IMessageUser
+    {
+        [JsonProperty("id")]
+        public long Id { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("screen_name")]
+        public string ScreenName { get; set; }
+
+        [JsonProperty("protected")]
+        public bool Protected { get; set; }
+
+        [JsonProperty("verified")]
+        public bool Verified { get; set; }
+
+        [JsonProperty("followers_count")]
+        public int FollowersCount { get; set; }
+
+        [JsonProperty("friends_count")]
+        public int FriendsCount { get; set; }
+
+        [JsonProperty("statuses_count")]
+        public int StatusesCount { get; set; }
+
+        [JsonProperty("profile_image_url")]
+        public string ProfileImageUrl { get; set; }
+
+        [JsonProperty("profile_image_url_https")]
+        public string ProfileImageUrlHttps { get; set; }
+
+        [JsonProperty("created_timestamp")]
```

Comment:
This is the only field that is different from the `UserDTO` class.
Also this field should be a `DateTime`

``` c#
[JsonProperty("created_timestamp")]
[JsonConverter(typeof(JsonTwitterDateTimeConverter))]
public DateTime CreatedTimestamp { get; set; }
```

Reply:
Hi Linvi,

You can reuse the code and do a different check in, that is ok for me, thanks!

/Rehan